hs_core/views/__init__.py

```diff
@@ -208,6 +211,15 @@ def update_metadata_element(request, shortkey, element_name, element_id, *args,
 
     return HttpResponseRedirect(request.META['HTTP_REFERER'])
 
+@api_view(['GET'])
+def file_download_url_mapper(request, shortkey, filename):
+    """ maps the file URIs in resourcemap document to django_irods download view function"""
+
+    authorize(request, shortkey, view=True, edit=True, full=True, superuser=True)
```

Comment:
I think we should leave in the call to authorize() here to signal intent, and also to protect against authorization regressions if we refactor our django_irods download mechanism.  If we notice performance problems, and profiliing shows the authorization call to be the culprit, then we can refactor to coalesce authorization calls.  Otherwise, we should error on the side of calling authorize() perhaps unnecessarily in some cases.

@hyi I agree that we should only include the api_view decorator if it is absolutely needed.  @pkdash if you've tested this with the REST API and it is needed, then we should keep it.  Otherwise, I would remove it for now and only add it later if we have a need.